src/rate.rs

```diff
@@ -747,7 +752,7 @@ impl RCState {
         // If this is the first example of the given frame type we've seen, we
         //  immediately replace the default scale factor guess with the
         //  estimate we just computed using the first frame.
-        if self.nframes[fti] <= 0 {
+        if trial || self.nframes[fti] <= 0 {
```

Comment:
Is there ever a time when only one of these is true? i.e. can they be merged?

Reply:
At present, trial implies `self.nframes[fti] <= 0` but my intention was to be explicit about the dual use of this block.

Reply:
That's what I thought. No complaint from me if you prefer it.